source/frontend/src/components/Table/PageSizeSelector.tsx

```diff
@@ -38,17 +33,25 @@ export const TablePageSizeSelector: React.FC<React.PropsWithChildren<IProps>> =
   }));
   return (
     <Menu options={pageSizeOptions} alignTop={alignTop}>
-      <div style={{ display: 'flex' }}>
+      <div className="d-flex">
         <StyledText>Show</StyledText>
-        <Form.Control
-          size="sm"
-          value={`${selected}`}
-          type="number"
-          style={{ width: 50, marginLeft: 10, marginRight: 10, backgroundColor: 'white' }}
-          disabled
-        />
+        <StyledInput size="sm" value={`${selected}`} type="number" disabled />
         <StyledText>Entries</StyledText>
       </div>
     </Menu>
   );
 };
+
+/** align text to middle of size input */
+const StyledText = styled.span`
+  margin-top: 0.3rem;
+`;
+
+const StyledInput = styled(Form.Control)`
+  width: 7.5rem;
```

Comment:
this appears to be 25ps bigger then it previously was.

Reply:
It is - we need to make it wider to display 2 and 3 digit numbers. The actual bug is that it says it "shows 1 instead of 10".
I found out that the component does "display" 10 by default but the input is too narrow so it only displays the first digit